setup-tektoncd/README.md

```diff
@@ -0,0 +1,122 @@
+[![use-action][useActionWorkflowBadge]][useActionWorkflow]
+
+`setup-tektoncd`
+----------------
+
+Action to rollout [Tekton Pipeline][githubTektonPipeline], [CLI (`tkn`)][githubTektonCLI] and a [Container-Registry][containerRegistry] instance, setting up the environment for testing using these components.
+
+# GitHub Action Usage
+
+Example usage below with inputs using default values:
+
+```yaml
+---
+jobs:
+  setup-tektoncd:
+    steps:
+      # using KinD to provide the Kubernetes instance and kubectl
+      - uses: helm/kind-action@v1.5.0
+        with:
+          cluster_name: kind
+
+      # setting up Tekton Pipelines, CLI and additional components...
+      - uses: tektoncd/actions/setup-tektoncd@main
```

Comment:
I'm not entirely sure this type of repository organisation is supported by GHA, but would be great to be able to ship more than one action per repository 🙏

Reply:
https://github.com/orgs/community/discussions/24990 -- the answer is "yes", a repository supports multiple actions 👍